src/modules/governance/pages/daoMemberDetailsPage/daoMemberDetailsPageClient.test.tsx

```diff
@@ -38,7 +38,7 @@ jest.mock('@/modules/governance/components/voteList', () => ({
     VoteList: jest.fn(() => <div data-testid="vote-list-mock" />),
 }));
 
-describe.skip('<DaoMemberDetailsPageClient /> component', () => {
+describe('<DaoMemberDetailsPageClient /> component', () => {
```

Comment:
Removed skip here as it was an artifact from another PR during offsite I just realized when testing all 'src' - all tests are passing.